### ISSUE TITLE
Uncaught Error: Invalid hook call.

### ISSUE THREAD

**Issue by cashlycash** [iamskok/use-font-face-observer#155]
Every time I use the function "useFontFaceObserver" I am struck with the error 
```Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
    at resolveDispatcher (react.development.js:1476:1)
    at useState (react.development.js:1507:1)
    at e (index.ts:54:1)
    at App (index.js:23:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:1)
    at invokeGuardedCallback (react-dom.development.js:4277:1)```

**Comment by emadmrz** [comment]
same error here !

**Comment by oleksandr-danylchenko** [comment]
I suppose it's related to the #147 issue. Unfortunately, installing 2 `react` libraries within the same project will lead to the exact error you see above. To fix it locally, you can try to use the `yarn`'s `resolutions that will specify just the `react` version that your project needs. E.g.:
```
"resolutions": {
	"@types/react": "^16.9.43",
	"@types/react-dom": "^16.9.8",
	"react": "^16.13.1",
	"react-dom": "^16.13.1"
},
```

**Comment by iamskok** [comment]
The fix was merged and released as v1.2.2
